packages/forms-web-app/package.json

```diff
@@ -34,6 +34,7 @@
     "@ministryofjustice/frontend": "0.2.0",
     "@pins/business-rules": "file:../business-rules",
     "@pins/common": "file:../common",
+    "@planning-inspectorate/pins-clamav-rest-client-rest-client": "^1.0.0",
```

Comment:
Looks too long? Should be `pins-clamav-rest-client`?

Reply:
Done